docs/src/pages/Conformance.md

```diff
@@ -248,6 +248,13 @@ Reference searches on the IBM FHIR Server support search on:
+* If a resource has a reference that is versioned, and a reverse chain search is performed using the element containing the versioned reference, then the referenced resource can only be returned as a match if the version specified is the referenced resource's current version. For example, if a `Condition` resource contains a reference of `Patient/123/_history/2` in its `subject` element, and the current version of the `Patient/123` resource is 2, and a search of `Patient?_has:Condition:patient:code=1234-5` is performed and the `Condition` resource meets the search criteria, then the `Patient/123` resource will be returned as a match since the version specified in the reference is the current version of the `Patient/123` resource. If the current version of the `Patient/123` resource was 3, then the resource would not be returned as a match in the search results. This is because the IBM FHIR Server will only return the current versions of `match` resources in search results.
+* If a resource has a reference that is versioned, and an `_include` search is performed using the element containing the versioned reference, then the version specified in the reference will be returned as an `include` resource in the search results. For example, if a `Condition` resource contains a reference of `Patient/123/_history/1` in its `subject` element, and the current version of the `Patient/123` resource is 2, and a search of `Condition?_include=Condition:subject` is performed, then version 1 of the `Patient/123` resource will be returned as an `include` resource in the search results.
+* If a resource has a reference that is versioned, and a `_revinclude` search is performed using the element containing the versioned reference, then the resource containing the versioned reference is returned as an `include` resource only if the version specified in the reference is the referenced resource's current version. For example, if a `Condition` resource contains a reference of `Patient/123/_history/2` in its `subject` element, and the current version of the `Patient/123` resource is 2, and a search of `Patient?_revinclude=Condition:subject` is performed, then the `Condition` resource will be returned as an `include` resource since the version of the `Patient` resource specified in the `subject` element is the current version of the `Patient` resource. If the current version of the `Patient/123` resource was 3, then the `Condition` resource would not be returned as an `include` resource in the search results. This is because the IBM FHIR Server will only return the current versions of `match` resources in search results, so a reference to a non-current version of the resource is not considered to have met the search criteria, thus the resource containing the reference is not considered a valid `include` resource.
```

Comment:
Similar recommendation on the example/presentation. It's so very dense.

Last sentence.  It's a mixed tense maybe focus on the present tense or future tense only.  `The IBM FHIR Server only returns the current version of`

Reply:
Fixed tense.